CHANGELOG.md

```diff
@@ -1,20 +1,9 @@
-1.9.11-edge.3 / 2023-07-03
+1.9.11 / 2023-07-16
 ==================
 
-* Fix: seek-thumbnail compatibility with other plugins
 * Add files & types to package.json
-
-
-1.9.11-edge.1 / 2023-06-26
-==================
-
-Changes
--------------
-- Turn seek-thumbnail on by default
-
-1.9.11-edge.0 / 2023-06-18
-==================
-
```

Comment:
@jakub-roch this shouldn't happen. could it be that you started from `master` instead of `edge` ?